ui/src/containers/NodePageMetricsTab.tsx

```diff
@@ -131,8 +133,14 @@ const NodePageMetricsTab = ({
     (state) => state.app.monitoring.nodeStats.showAvg,
   );
 
-  // Create color set for this single node
-  const colorSet = useChartColors([nodeName]);
+  // Create color set
+
+  const colorSet = {
+    [nodeName]: lineColor1,
+  };
+  if (showAvg) {
+    colorSet[CLUSTER_AVERAGE] = lineColor1;
+  }
```

Comment:
We display a dashed line for the average value today. I think we don't have dash line in the new linetimeseriechart.

<img width="652" height="472" alt="Image" src="https://github.com/user-attachments/assets/1c1c0a4c-3d9b-4709-9cd9-0151d7e72761" />

Reply:
Yes let's add this dash line in linetimeserie.